src/serializer/serializer.js

```diff
@@ -175,18 +175,24 @@ export class Serializer {
         );
 
         if (this.options.heapGraphFormat) {
+          if (additionalFunctionValuesAndEffects.size > 0) {
+            let diagnostic = new CompilerDiagnostic(
+              "Using __optimize with --heapGraphFilePath flag is not currently supported",
+              undefined,
```

Comment:
Wasn't sure what to give these fields since they're not really applicable. Suggestions?